spec/classes/init_spec.rb

```diff
@@ -467,7 +467,7 @@
       context "specified as #{value}" do
         let(:params) { { :enable_stats => value } }
 
-        it { should contain_file('ntp_conf').with_content(/^statsdir \/var\/log\/ntpstats\/$/) }
+        it { should contain_file('ntp_conf').with_content(/statsdir \/var\/log\/ntpstats\/.statistics loopstats peerstats clockstats.filegen loopstats file loopstats type day enable.filegen peerstats file peerstats type day enable.filegen clockstats file clockstats type day enable/m) }
```

Comment:
The 'm' after the regexp allows multiple lines and the dot stands for the
'\n'. Not sure how '^' and '$' behave here, but I can try tomorrow
Am 31.08.2015 20:08 schrieb "Garrett Honeycutt" <notifications@github.com>:

> In spec/classes/init_spec.rb
> <https://github.com/ghoneycutt/puppet-module-ntp/pull/82#discussion_r38340352>
> :
>
> > @@ -467,7 +467,7 @@
> >        context "specified as #{value}" do
> >          let(:params) { { :enable_stats => value } }
> >
> > -        it { should contain_file('ntp_conf').with_content(/^statsdir \/var\/log\/ntpstats\/$/) }
> > +        it { should contain_file('ntp_conf').with_content(/statsdir \/var\/log\/ntpstats\/.statistics loopstats peerstats clockstats.filegen loopstats file loopstats type day enable.filegen peerstats file peerstats type day enable.filegen clockstats file clockstats type day enable/m) }
>
> could you explain this regex?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ghoneycutt/puppet-module-ntp/pull/82/files#r38340352>.
>